Neos.Media/Classes/Domain/Model/Adjustment/ResizeImageAdjustment.php

```diff
@@ -76,7 +76,7 @@ class ResizeImageAdjustment extends AbstractImageAdjustment
      * @var string
      * @ORM\Column(nullable = true)
```

Comment:
I guess the nullable true could then be removed here?

Reply:
No, because the table is shared with all the other adjustment, and they do not have the property…

Reply:
True, and the migration needed corresponding adjustment

Reply:
Don't get that… "would need" or "did need"? And what would you use as default if this was non-null? Having e.g. `inset` in every entry that is not a `ResizeImageAdjustment` seems counter-intuitive.

Reply:
would need, sorry :)
> And what would you use as default if this was non-null?

No default, it has to be specified (and with the new code it _is_)

Reply:
Exactly, I missed the part about the shared table, that makes sense ofc,,,